src/overlays/actors/ovl_Demo_Du/z_demo_du.c

```diff
@@ -1,228 +1,1044 @@
 #include "z_demo_du.h"
+#include "../ovl_Demo_Effect/z_demo_effect.h"
```

Comment:
we dont do relative paths for includes so youll want to start from the `overlays/` folder